components/Layout/Navbar/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+function Navbar() {
+    const items = ['Über uns', 'Wie wir arbeiten', 'Services', 'Projekte', 'Branchen', 'Karriere', 'What’s up?']
+  return (
+    <div>
+        <div className="flex justify-between bg-secondary text-white px-6 py-3 fixed top-0 left-0 right-0">
+            <div className="flex items-center gap-8">
+                <div className="text-[23px]">IBM <strong>iX</strong></div>
```

Comment:
Try to use rem and define this specifically by extending the TailwindCSS configuration and remove this arbitrary values.